app/assets/stylesheets/chr/themes/basic.scss

```diff
@@ -117,27 +117,24 @@ a {
 }
 
 .header-tabs {
-  border : 1px solid $positive-color;
+  border: 1px solid $positive-color;
+  background-color: $positive-color;
 
-  a {
+  button {
+    font-size: 0.8em;
```

Comment:
Properties should be ordered background-color, border, color, font-size, margin-right, padding